src/styles/style.scss

```diff
@@ -16,4 +16,13 @@
 @import "_colorschema_light";
 @import "_colorschema_dark";
 
-@import "alignment";
\ No newline at end of file
+@import "alignment";
+
+body {
+    -webkit-touch-callout: auto;
```

Comment:
do we really want these styles unconditionally on the body tag? This could impact the underlying site.

Reply:
It is an interesting question:
if site doesn't allow to select a word - our extension couldn't work properly. As we use seletions.
And site from the issue has these styles in tag - `user-select: none`
Chrome and Safari have limited problems with it (it is difficult to properly select on such pages), Safari couldn't handle with selection in this case.

So we should decide - how to handle here.
I suggest to simply add these styles - as if someone activated an extension - then someone wants to be able to select, I think.